drivers/sensor/lsm303dlhc_magn/Kconfig

```diff
@@ -0,0 +1,69 @@
+# Kconfig - LSM303DLHC Three Axis Magnetometer configuration options
+
+#
+# Copyright (c) 2018 Philémon Jaermann
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig LSM303DLHC_MAGN
+	bool
+	prompt "LSM303DLHC Three Axis Magnetometer"
+	depends on I2C
+	default n
+	help
+	  Enable driver for LSM303DLHC I2C-based triaxial magnetometer sensor.
+
+config LSM303DLHC_MAGN_NAME
```

Comment:
Remove the name, i2c master dev, and i2c address configs, and get them from device tree instead.

Reply:
Done